README.md

```diff
@@ -418,34 +452,40 @@ This ensures that there are no unwanted configuration changes that would be resu
 Start by creating a netsim device that you will configure with loopback interfaces in the test. Create new folder `test` and use `ncs-netsim` command to add a ios-xr device to it.
 
 ```
-mkdir test
+mkdir ~/src/test
 cd test
```

Comment:
can you add the relative path? this will ensure users go to the right dir no matter where they were.
```bash
cd ~/src/test
```